contributors/devel/e2e-tests.md

```diff
@@ -580,6 +569,21 @@ breaking changes, it does *not* block the merge-queue, and thus should run in
 some separate test suites owned by the feature owner(s)
 (see [Continuous Integration](#continuous-integration) below).
 
+  - `[Conformance]`: Designate that is test is included in the Conformance
+test suite for [Conformance Testing](conformance-tests.md). This test must
+meet a number of [requirements](conformance-tests.md#conformance-test-requirements)
+to be eligible for this tag. This tag does not supersed any other labels.
+
+  - The following tags are not considered to be exhaustively applied, but are
```

Comment:
I'm not sure what the point of these tags are since they seem to be orthogonal to conformance testing. From this doc's POV we should focus just on what it takes to be a conformance test. If people want to add tags for other purposes (e.g. to know which tests are 'alpha') then we should talk about that in some other testing doc. Talking about it here is confusing and could be misleading , for example, it could imply conformance tests might be related to "alpha" test/features when we specifically say above that they're not. I'd suggest removing this.

Reply:
@duglin we need this info to later organize the stuff into `buckets`. we need these to live in the test case themselves.

Reply:
@dims during the last sig-arch call we did agree to determine what the categorized of "other" tests are so that we can have a discussion about what to do with them. However, I think its premature to then put text in here that might encourage people to open PRs to add these tags to our tests. I say this because, 1) we haven't agreed on what the tags should be, 2) after the analysis of these categories we may decide that each of them can be merged into the single conformance test suite that we have, thus then requiring follow-on PRs to remove these conformance-wise unnecessary tags.  As of now, we only have one tag, Conformance, and that's the only tag we should mention until we have further discussions in the conformance SIG.

If there's a desire to include something about this "other" stuff, I would prefer something that describes our current plans, something perhaps like:
- - -
There are certain features that are relatively easy to classify as a natural fit for our conformance test suite, such as CRUD operations on Pods. However, there are some features in Kubernetes that are less clear, often due to the variability, or optionality, of their support in certain implementations. SIG-Conformance is under-going an analysis of these features to determine the best path forward to determine if, or how, to include them within the conformance test suite.
- - -

In the meantime, we should gather the list of potential categories (the list in this PR is a good start) in a google doc, encourage people to add more (if needed) along with a justification for why they should, or should not, be part of conformance. Then we can discuss it in the next conformance meeting.

Reply:
Only reason I suggested a google doc is because its easier for quick adds/removes of ideas. PR'ing something as part of a brainstorming/analysis process (which is the phase I think we're in) seems kind of heavy-weight.  I would also like us to go to weekly conformance SIG meetings for this very reason - I've added it to the agenda for our next call. Monthly meetings, IMO, discourage the community aspect of our SIG - which relates to the next point.

re: SIG-Conformance vs SIG-Arch... I think we probably need to get some clarity on process here. I definitely agree that whatever we do for conformance needs to be approved by SIG-Arch, but if all of our day-to-day brainstorming is done within the context of SIG-Arch then we might as well shutdown SIG-Conformance.  I would prefer a process, like all other SIGs, where we work on stuff and report our decisions up to SIG-Arch - and in some cases report them up for approval and not just when we need a big decision made. Granted we may get more oversight than some other SIGs due to the nature of our work, but I do want us to feel like we can have some degree of independence.

Reply:
The way i see it. this document is providing guidelines from sig-arch for the day to day work in the conformance sig.

Reply:
I can't find this section in https://docs.google.com/document/d/1kwsS0d0cVTg4idIOGrWzNHBoZ3UyEGi0fELbBlADgmI/edit  so I'm not sure its fair to imply that its been iterated on and agreed to.

Reply:
Sourced from https://docs.google.com/document/d/1kwsS0d0cVTg4idIOGrWzNHBoZ3UyEGi0fELbBlADgmI/edit?disco=AAAACLaeHK4

Reply:
yup- same doc I pointed to.  No mention of this section in there.

Reply:
@duglin This was raised in comments on that document and in 2 SIG Architecture meetings.

It is not just about "other" tests. Some tests that are currently part of the conformance suite are believed to not adhere to the aspirational criteria set forth by this doc. We want to assess how many tests fall into those categories and tag the tests so that they don't have to be re-identified later.

Reply:
From a process perspective I would prefer that text being proposed, reviewed and implicitly agreed to, be put into the body of the document for each review and not left in a comment where it is easily missed - or interpreted as a WIP thought exercise.

Reply:
Let's start with this list, and if we find it's too granular, or not granular enough, we can amend the list